main_timer_rate: 30.0
swarm_timer_rate: 30.0

tracking:
  radius: 8.0
  speed: 3.0
  height: 2.0
  trajectory_steps: 50

searching:

  initial_position:
    x: 0.0
    y: 0.0

  radius: 25.0
  speed: 5.0
  height: 4.0
  trajectory_steps: 50
  heading_rate: 0.3
